### PR TITLE
move CDN module to new repo

### PR DESCRIPTION
**Pull request by domjngo**

### CODE REVIEW

File: cloudfront.tf [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    tags = {
+        Name        = "${var.cdn_bucket_name}-cloudfront"
+        Environment = var.environment
+        Service     = var.service
+        Owner       = var.owner
+        CreatedBy   = var.created_by
+        CostCentre  = var.cost_centre
+        Terraform   = "True"
```
Comment (karlkern) [inline cloudfront.tf]: Could we change the values to lower case?

File: s3.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    }
+
+    tags = {
+        Name        = var.cdn_bucket_name
+        Environment = var.environment
+        Service     = var.service
+        Owner       = var.owner
+        CreatedBy   = var.created_by
+        CostCentre  = var.cost_centre
+        Terraform   = "True"
```
Comment (karlkern) [inline s3.tf]: Lower case?

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+ij_python_spaces_around_shift_operators = true
+ij_python_spaces_within_braces = false
+ij_python_spaces_within_brackets = false
+ij_python_spaces_within_method_call_parentheses = false
+ij_python_spaces_within_method_parentheses = false
+ij_python_use_continuation_indent_for_arguments = false
+ij_python_use_continuation_indent_for_collection_and_comprehensions = false
+ij_python_wrap_long_lines = false
+
+[{*.tf,*.tfvars}]
```
Comment (karlkern) [inline .editorconfig:270]: hyphen used. Is this on purpose as others have only underscores?

Comment (domjngo) [reply .editorconfig:270]: I haven't changed this. This is exactly as given.

**Review by karlkern — APPROVED** [review]

**Pull request merged**